src/main/java/seedu/address/storage/XmlAdaptedPerson.java

```diff
@@ -29,6 +30,8 @@
     private String email;
     @XmlElement(required = true)
     private String address;
+    @XmlElement(required = true)
```

Comment:
should this be required?

Reply:
yeah, this field is required....
maybe that's why you met a problem adding a blindListener